### ISSUE TITLE
Audiobook cli only

### ISSUE THREAD

**Issue by crozo5** [subdavis/kobo-book-downloader#11]
What means cli only?? I can download my normal books, but I can't download my audiobooks.
How can I download my audiobooks??

**Comment by Madam-Mim** [comment]
CLI = command line interface. Currently books can be downloaded via both the cli and from a browser page generated by the built in server. Audio books only from the command line version at the moment. I've not tried it yet on this fork. I only have one audio book and that was downloaded with a development of the original TnS-hun Kobo Book Downloader.

**Comment by crozo5** [comment]
I have already managed to download an audiobook and everything is correct, the only problem is that it is divided into several mp3s of more or less 1 hour duration.

Would it be possible to download the audiobook dividing it by chapters?

**Comment by subdavis** [comment]
No.  The way they get downloaded is how they come from Kobo.  It might be possible to download the associated metadata and use a different tool to re-splice the audio, but I don't know if I'm interedted in spending time on that.

Leaving this issue open until documentation is improved for what "CLI only" means.

**Comment by subdavis** [comment]
Documentation added.
